tests/pytorch/test_config_validator_pytorch.py

```diff
@@ -0,0 +1,29 @@
+# Future
+from __future__ import print_function
+
+# Standard Library
+import os
+from unittest.mock import patch
+
+# Third Party
+import pytest
+
+# First Party
+import smdebug.pytorch.singleton_utils
+from smdebug.core.config_validator import reset_config_validator
+
+
+@pytest.fixture(autouse=True)
+def cleanup():
+    smdebug.pytorch.singleton_utils.del_hook()
```

Comment:
nit: import the function directly and call the function

```
from smdebug.pytorch.singleton_utils import del_hook
...
del_hook()
```

Reply:
Please do this everywhere in the tests (to reduce code redundancy)

Reply:
done